package.json

```diff
@@ -7,14 +7,14 @@
   "main": "index.js",
   "scripts": {
     "build": "webpack --config prod.config.js -w",
-    "start": "node server/",
-    "seed": "node server/scripts/seed",
+    "start": "nodemon server/",
+    "seed": "node server/database/seed",
     "test": "jest server/test/ --detectOpenHandles"
   },
   "dependencies": {
     "axios": "^0.19.2",
-    "body-parser": "^1.19.0",
     "css-loader": "^3.5.3",
+    "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "faker": "^4.1.0",
     "jquery": "^3.5.1",
```

Comment:
You changed the use of jQuery to Javascript's built-in `fetch`, so you could uninstall jQuery as a dependency.

Reply:
Good point - will fix before merging!